### PR TITLE
[docs]: document sync for spiderpool

### PR DESCRIPTION
**Pull request by ty-dc**
Some documents are distributed in the public account and Twitter, and they are organized and integrated into the spiderpool in this pr.

**make sure your commit is signed off**
Signed-off-by: ty-dc [tao.yang@daocloud.io](mailto:tao.yang@daocloud.io)

**Please wait for the review, we need to supplement this month's documentation and optimize other documents that use the veth plugin**

### CODE REVIEW

File: docs/usage/ip-allocation-across-networks-zh_CN.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
+# 云原生 Spiderpool：基于跨越网络区域的 IP 分配
+
+Underlay 网络是底层基础网络，具有低延迟、可靠、安全等特性。这些特性能满足一些延时敏感的应用业务需求、提供优质的用户体验以及有效管理和保护网络资源，通常应用在延时敏感、防火墙安全管控中：
```
Comment (Michelle951) [inline docs/usage/ip-allocation-across-networks-zh_CN.md:3]:
```suggestion
Underlay 网络是底层基础网络，具有低延迟、可靠、安全等特性。这些特性能满足一些延时敏感的应用业务需求、提供优质的用户体验以及有效管理和保护网络资源，通常应用在延时敏感、防火墙安全管控中。
```

File: docs/usage/ip-allocation-across-networks-zh_CN.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+  gateway: 10.7.0.1
+  nodeAffinity:
+    matchExpressions:
+    - {key: node-subnet, operator: In, values: [subnet-7]}
+EOF
+```
+
+## 创建应用
+
+以下的示例 Yaml 中， 会创建一个 daemonSet 应用，其中：
```
Comment (Michelle951) [inline docs/usage/ip-allocation-across-networks-zh_CN.md:129]:
```suggestion
以下的示例 Yaml 会创建一个 daemonSet 应用，其中：
```

File: docs/usage/ip-allocation-across-networks.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
+# Cloud-Native Spiderpool: IP Allocation Across Network Zones
+
+An underlay network, as the name implies, is an underlying network that features low latency, high reliability, and strong security. These characteristics are exactly  what latency-sensitive applications need for optimal user experience, and also facilitate efficient management and protection of network resources. Underlay networks are commonly used for latency-sensitive applications, firewall security control and other scenarios.
```
Comment (Michelle951) [inline docs/usage/ip-allocation-across-networks.md:3]:
```suggestion
An underlay network, as the name implies, is an underlying network that features low latency, high reliability, and strong security. These characteristics are exactly  what latency-sensitive applications need for optimal user experience, and also facilitate efficient management and protection of network resources. Underlay networks are commonly used for latency-sensitive applications and firewall security control.
```

**Review by Michelle951 — APPROVED** [review]

**Comment by weizhoublue** [comment]
这些图 都带了 背景色，应该是 软件没 export  为 jpg，

**Comment by windsonsea** [comment]
是的，这些图片像是 screenshot，需要导出或另存为吧

**Comment by weizhoublue** [comment]
@ty-dc it seems not not update for long time

**Comment by ty-dc** [comment]
与 #2046 存在冗余，以 2046 为准。

**Pull request closed without merging**